package.json

```diff
@@ -13,7 +13,7 @@
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-prettier": "^3.1.1",
     "eslint-plugin-react": "^7.14.3",
-    "eslint-plugin-react-hooks": "^2.0.1",
+    "eslint-plugin-react-hooks": "^1.7.0",
```

Comment:
Why the downgrade?

Reply:
Fordi regelpakke eslint-config-airbnb krever ^1.7, og det finnes ikke ny version av dette enda.